rajawali/src/main/java/c/org/rajawali3d/sceneview/SceneView.java

```diff
@@ -0,0 +1,227 @@
+package c.org.rajawali3d.sceneview;
+
+import android.graphics.Rect;
+import android.opengl.GLES20;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import c.org.rajawali3d.annotations.GLThread;
+import c.org.rajawali3d.annotations.RequiresReadLock;
+import c.org.rajawali3d.camera.Camera;
+import c.org.rajawali3d.engine.Engine;
+import c.org.rajawali3d.engine.RenderModel;
+import c.org.rajawali3d.engine.RenderView;
+import c.org.rajawali3d.object.renderers.ObjectRenderer;
+import c.org.rajawali3d.scene.Scene;
+import c.org.rajawali3d.scene.graph.NodeMember;
+import java.util.List;
+import java.util.concurrent.locks.Lock;
+import net.jcip.annotations.GuardedBy;
+import org.rajawali3d.math.Matrix4;
+import org.rajawali3d.util.RajLog;
+
+/**
+ * Author: Randy Picolet
+ */
+
+public class SceneView implements RenderView {
+    private static final String TAG = "SceneView";
+
+    @Nullable
+    private Engine engine;
+
+    @NonNull
+    private Scene scene;
+
+    // TODO probably need a decorator Viewport class around the Rect to flag changes...
```

Comment:
I'll throw the basic decorator together, then we can add stuff as we think of it